src/interface/variable.hpp

```diff
@@ -295,12 +303,13 @@ class ParticleVariable {
   /// return information string
   std::string info() const;
 
-  ParArrayND<T> data;
-
  private:
-  int npool_;
   Metadata m_;
   std::string label_;
+  std::array<int, 6> dims_;
+
+ public:
+  ParArrayND<T> data;
```

Comment:
As for the FaceFields, why is this change required?
I'm asking as this is a parthenon public interface.

Reply:
It's an initializer list problem, for `ParticleVariable`s it makes more sense to initialize `data` after `dims_`. Does that seem like an OK change?